translation/source/puzzleTheme.xml

```diff
@@ -119,8 +119,8 @@
   <string name="xRayAttackDescription">A piece attacks or defends a square, through an enemy piece.</string>
   <string name="zugzwang">Zugzwang</string>
   <string name="zugzwangDescription">The opponent is limited in the moves they can make, and all moves worsen their position.</string>
-  <string name="healthyMix">Healthy mix</string>
-  <string name="healthyMixDescription">A bit of everything. You don't know what to expect, so you remain ready for anything! Just like in real games.</string>
+  <string name="mix">Healthy mix</string>
+  <string name="mixDescription">A bit of everything. You don't know what to expect, so you remain ready for anything! Just like in real games.</string>
```

Comment:
this will ping all translators to ask them for a new translation for `mix` and `mixDescription`.

Reply:
Yep, we'll have to re-tag the screenshot too.

Crowdin's Translation Memory (TM) will help translators re-use the previous string. Ideally we could programmatically migrate all the XML files under `translation/dest` so existing localisations aren't lost. I suspect this updates the string from the Lichess account on Crowdin, but would have to test it to be sure. I am basing this on a past update Thibault did a few years ago.

If it *doesn't* update backwards, we would still lose them in Crowdin but can fix that manually by just re-adding the translation to every language.

Reply:
So for clarity - if we DON'T transition translations to `mix` and `mixDescription`, just keep the old strings around in the source XML instead of renaming.

Reply:
How long should we keep the old string? Surely we don't want to keep it forever.